emcc.py

```diff
@@ -395,6 +395,23 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  apply_min_browser_versions(user_settings)
+
+
+def default_min_browser_version(user_settings, browser, version):
+  default_setting(user_settings, f'MIN_{browser.upper()}_VERSION', version)
```

Comment:
Maybe make this a local function within apply_min_browser_versions? (then you don't need to pass user_settings to it).